src/compiler/transformers/es2015.ts

```diff
@@ -4417,4 +4421,40 @@ namespace ts {
             return isIdentifier(expression) && expression.escapedText === "arguments";
         }
     }
+
+    function forEachFreeIdentifier(node: Node, cb: (id: Identifier) => void): void {

```

Comment:
I've copied this one from `src/services/codefixes/convertToEs6Module.ts`. What will be the best place to put it so it can be shared?

Reply:
`src/compiler/utilities.ts`

Reply:
@andrewbranch thanks, done